README.md

```diff
@@ -1,4 +1,4 @@
-# MSiA 423 Project - CuisineHelper
+# MSiA 423 Project - CuisineHelpr
```

Comment:
Is the CuizineHelpr spelled without an E?

Reply:
It's branding